src/commands/run.js

```diff
@@ -13,7 +13,7 @@ const sendLoadingMessage = async (credentials) => {
 
 const getReportMessage = async (credentials, url) => {
   const reportURL = await generateFullReport(credentials, url);
-  return `You can view your report here: ${reportURL}`;
+  return `You can view your report for ${url} here: ${reportURL}`;
```

Comment:
Can we add do this instead:
```
`You can view your report for \`${url}\` here: ${reportURL}`
```
So we only have 1 link in the message instead of 2!